components/core/JsInterop/DomEventService.cs

```diff
@@ -1,13 +1,25 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Linq;
 using System.Text.Json;
 using Microsoft.JSInterop;
 
 namespace AntDesign.JsInterop
 {
     public class DomEventService
     {
-        private Dictionary<string, Func<object>> _domEventListeners = new Dictionary<string, Func<object>>();
+        private object _subscriptionsLock = new object();
+        private Dictionary<string, List<Subscription>> _domEventSubscriptions = new Dictionary<string, List<Subscription>>();
+        private Dictionary<string, List<Subscription>> DomEventSubscriptions
+        {
+            get
+            {
+                lock (_subscriptionsLock)
```

Comment:
What's the purpose of this lock? You're locking over the getter and this won't safeguard any operations on the dictionary.

Reply:
How about use `CconcurrentDictionary`?